contracts/Whitelist.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+import "./packages/oz/Ownable.sol";
+
+/**
+ * @author Opyn Team
+ * @title Whitelist Module
+ * @notice The whitelist module keeps track of all valid Otoken contracts.
+ */
+contract Whitelist is Ownable {
+    mapping(bytes32 => bool) internal whitelistedProduct;
+
+    event ProductWhitelisted(
+        bytes32 productHash,
+        address indexed underlying,
+        address indexed strike,
+        address indexed collateral
+    );
+
+    /**
+     * @notice check if a product is supported
+     * @dev product = the hash of underlying, collateral and strike asset
+     * @param _underlying option underlying asset address
+     * @param _strike option strike asset address
+     * @param _collateral option collateral asset address
+     * @return boolean, true if product is supported
```

Comment:
When are we using _name vs name? @antoncoding thoughts?

Reply:
Can we define that clearly? Just so from a style perspective its all good?

Reply:
_name for function args and for any internal/private state variable.

Reply:
I'm using `_` for all private variables and internal functions.